### ISSUE TITLE
Sediment deposition progress logging not advancing linearly

### ISSUE THREAD

**Issue by phargogh** [natcap/invest#344]
A [user on the forums](https://community.naturalcapitalproject.org/t/the-running-of-sdr-invest-3-8-2-get-stuck-or-closed-and-result-is-not-complete/1415) recently posted the attached logfile (from on InVEST 3.8.2), and you can see that sediment deposition logging is not increasing predictably and linearly over time, it's jumping around a bit.  Here's an excerpt:

```
2020-10-12 12:12:58,553 Task._call(1069) INFO 4.08% complete
2020-10-12 12:12:58,557 Task._call(1069) INFO 4.21% complete
2020-10-12 12:12:58,560 Task._call(1069) INFO 4.34% complete
2020-10-12 12:12:58,564 Task._call(1069) INFO 4.47% complete
2020-10-12 12:12:58,566 Task._call(1069) INFO 4.49% complete
2020-10-12 12:12:58,570 Task._call(1069) INFO 0.25% complete
2020-10-12 12:12:58,574 Task._call(1069) INFO 0.51% complete
2020-10-12 12:12:58,578 Task._call(1069) INFO 0.77% complete
2020-10-12 12:12:58,582 Task._call(1069) INFO 1.02% complete
2020-10-12 12:12:58,586 Task._call(1069) INFO 1.28% complete
2020-10-12 12:12:58,590 Task._call(1069) INFO 1.53% complete
2020-10-12 12:12:58,594 Task._call(1069) INFO 1.79% complete
2020-10-12 12:12:58,598 Task._call(1069) INFO 2.04% complete
2020-10-12 12:12:58,602 Task._call(1069) INFO 2.30% complete
2020-10-12 12:12:58,606 Task._call(1069) INFO 2.56% complete
2020-10-12 12:12:58,637 Task._call(1069) INFO 2.81% complete
```

I don't know enough about the sediment deposition routine to say for certain if this is expected behavior, but it sure _looks_ unexpected in the logfile.

Also, while we're in there, could we update the progress message to be `"calculate sediment deposition %s%% complete"`?  Since it isn't possible to display the function name in the logs when the function is a cython function, it would be helpful to have a little more information in progress logging about what's progressing.

[1.1-2015-lake-ok】InVEST-Sediment-Delivery-Ratio-Model-(SDR)-log-2020-10-12--12_07_04.txt](https://github.com/natcap/invest/files/5393528/1.1-2015-lake-ok.InVEST-Sediment-Delivery-Ratio-Model-.SDR.-log-2020-10-12--12_07_04.txt)

**Comment by phargogh** [comment]
Working on this in my fork at `bugfix/344-sediment-deposition-progress-logging`.

**Comment by emlys** [comment]
Closed in #372
